### ISSUE TITLE
[7.3.2-production] Safari HLS playback issue with preload: none

### ISSUE THREAD

**Issue by pixelchutes** [jwplayer/jwplayer#1071]
### Expected Behavior

Playback of HLS media should work _automatically_ in Safari / OS X, even when using `preload: "none"`.

**NOTE:** This works as expected in `v7.2.4`, I believe this behavior was introduced in `7.3.0`

### Actual Behavior

Forever stuck in `buffering` state, playback requires manual intervention.
Removing `preload` directive allows for automatic playback...

**NOTE:** Only reproducible when HLS is not 1st item in playlist.

### Steps to reproduce

1. Load an HLS URL (in my case, over `https`) to JWPlayer `7.3.2` via `playerInstance.playlistItem(1)`;
    * This is in Safari 9.0.2 on OS X 10.11.2, where HLS is the 2nd item in the playlist.
2. Player enters `buffering` state, but playback never begins _(progress bar shows 00:00 of 00:00)_.
3. Controls show "pause" icon, clicking it results in `NaN:NaN`, like this: ![nan_safari](https://cloud.githubusercontent.com/assets/352182/13295181/d57ff162-dae4-11e5-95ec-32862d90a039.jpg)
4. Controls now show "play" icon, clicking it loads the HLS stream and playback begins. *(Manual intervention `playReason: "interaction"` required)*

### Code to reproduce
```html
<div id="player"></div>
<script>
    var demoPlaylist = [
        {
          title: 'First playlist item',
          file: 'https://content.jwplatform.com/videos/HkauGhRi-640.mp4',
          image: 'https://content.jwplatform.com/thumbs/HkauGhRi-640.jpg'
        },
        {
          title: 'HLS Video 1 (2nd playlist item)',
          description: 'This fails to start automatically in Safari / OS X',
          file: 'https://content.jwplatform.com/manifests/IMUjQRAB.m3u8',
          image: 'https://assets-jpcust.jwpsrv.com/thumbs/IMUjQRAB-720.jpg'
        }
    ];
    
    var playerInstance = jwplayer('player');
    playerInstance.setup({        
        playlist: demoPlaylist,
        preload: 'none', // Causes HLS to fail in Safari (if not first playlist item)        
        width: '100%',
        aspectratio: '16:9',
        visualplaylist: true,
        displaytitle: true,
        repeat: false        
    })
</script>
```
### Environment

- MacBook Pro
- OS X 10.11.2 / Safari 9.0.2
- JW Player 7.3.2

**Comment by robwalch** [comment]
Thanks Mike!

I have a fix for dev intended for 7.4. It's a bit more involved than what we want to change to fix this in 7.3. I'll have a simpler fix and patch for 7.3.2 shortly. We'll have a 7.3.3 up by EOD.

**Comment by pixelchutes** [comment]
:+1: Thank you!
